client/src/redux/sagas/datafiles.sagas.js

```diff
@@ -284,6 +284,12 @@ export function* renameFile(action) {
       payload: { status: 'SUCCESS', operation: 'rename' }
     });
     yield call(action.payload.reloadCallback, response.name, response.path);
+    yield put({
+      type: 'ADD_TOAST',
+      payload: {
+        message: `${file.name} renamed to ${action.payload.newName}`
```

Comment:
The current message scheme has some truncation logic lost here: https://github.com/TACC/Core-Portal/blob/main/client/src/components/DataFiles/DataFilesStatus/DataFilesStatus.js#L34

Perhaps for now we should still pass through file operations there, until we refactor the DataFilesStatus component

Reply:
I'll move the truncation logic into the batched messages. I tried integrating this into our existing datafiles notification scheme at first and it was a total mess, the interface is super convoluted and mixes in backend concerns.

Reply:
@jarosenb Fair enough, sounds good.